common_examples/iperf/README.md

```diff
@@ -0,0 +1,14 @@
+# Iperf example
+
+This exmaple provides a simple way to measure network performance using iperf.
+
+## About the example
+The example uses `iperf` and `iperf-cmd` components for an iperf implementation and providing command line interface for it. It provides DHCP server functionality for connecting to another ESP32, instead of a PC.
+
+## Configuring the example
+Using `idf.py menuconfig` setup the Ethernet configuration, in `Example option` you can enable DHCP server. 
+
+**Important**: If DHCP server functionality is enabled only first PHY can be used. In DHCP client mode you can use any available.
```

Comment:
Is this restriction really needed? I have an impression that ESP-IDF supports multiple DHCP Server instances. Please investigate.

Reply:
I will look into it as a separate commit into dev branch.

Reply:
I didn't want to introduce it outright because I was cautious about multiple running dhcp serves and how will both iperf and tcp_server work with it, how to setup IP ranges for two servers so the same IP is not set to different targets, and after that I decided to leave that out since the example is about iperf and not about DHCP server configuration.

Reply:
Introduced in all affected examples (tcp server and iperf).